src/timer.rs

```diff
@@ -21,7 +21,7 @@ pub fn interval_mod_setup(
     donation_conf: &DonationConfig,
 ) -> (u64, Option<u64>) {
     if donation_conf.percentage >= DONATION_THRESHOLD && !worker_conf.auto_tune {
-        return (100 * 60, Some(1));
```

Comment:
Unfortunately no. If auto tune is enabled it will regularly trash the threads anyway and start new. If donation hashing is enabled it will "sandwich" donation hashing between the bandit arm changes to not unnecessary disturb a running computation.
If there is no auto-tune the 100*60 ensures that every 100 minutes a change will occur and it then switches to donation hashing. It is quite a lot linked to how the banding arm changes...